ONSdigital/dp-census-atlas
# Map styling

### What

Improved base map style for clarity, and added white outline to black area selection outline.

### How to review

- View map to make sure style still loads correctly, and selected area has a white outline.
- View diffs. Changes only in two files.

### Who can review

Anyone can review.
